server/tests/helpers/documentsHelper.js

```diff
@@ -0,0 +1,41 @@
+const faker = require('faker');
+
+const documentsHelper = [
```

Comment:
This can be refactored to remove duplication, and so can the [userHelper](https://github.com/andela-pbirir/CP2-DMS/pull/8/files#diff-2b50acaa212ae29eb87a3a439e8a6074) and [roleHelper](https://github.com/andela-pbirir/CP2-DMS/pull/8/files#diff-9bf39382c9aa05204bbbfcfa03ae9e45).